### DISCUSSION TITLE
Change file directory option?

### DISCUSSION THREAD

**Discussion by cars11** [stashapp/stash#2063] [Q&A] [answered]
Am I correct in understanding that there is no way to change the directory of files short of editing the database?
(I moved a bunch of files to a new folder & drive). Do I have to delete them all and re-generate phashes etc?
I cannot find anyway to edit the directory path for anything

**Comment by bnkai** [comment, marked as answer]
Just do a new scan
Stash will find the new files calculate the checksums and update just the path when needed
Provided of course that the new folder is in your stash library. If it is not then add the directory first to the library
